tables/sw-ke-g1-2.ctb

```diff
@@ -0,0 +1,105 @@
+# liblouis: Swahili (Kenya) (Grade 1.2)
+#
+#-index-name: Swahili (Kenya) Grade 1.2
+#-display-name: Swahili (Kenya) Grade 1.2
+#
+#-author-name: Jake Kyle
+#-maintainer-name: Jake Kyle
+#-maintainer-email: jake@compassbraille.org
+#-updated: 2022
+#
+#+language:sw
+#+region:ke
+#+type:literary
+#+grade:1.2
```

Comment:
All the tables are missing `contraction` metadata. The grade 1 table should have `#+contraction: no`, the grade 1.2 to 1.5 tables should have `#+contraction: partial` and the grade 2 table shoudl have `#+contraction: full`.